src/components/Kv/KvLightbox.vue

```diff
@@ -103,6 +103,9 @@ export default {
 			}
 		}
 	},
+	mounted() {
```

Comment:
This just makes it so the lightbox can be immediately activated by passing true to the prop, since watch won't activate if initial value is true